### DISCUSSION TITLE
Issue Labels may be split into two

### DISCUSSION THREAD

**Discussion by ghost** [community/community#15746] [Mobile] [answered]
Hi there 😊

It appears as if a single issue label may be split into two on iOS if the label does no longer fit into one line.

Personally, I find this confusing as on first glance those seem to be individual labels.

Attached is a screenshot demonstrating the problem with this [element-ios issue](https://github.com/vector-im/element-ios/issues/6074).

![image](https://user-images.githubusercontent.com/61202918/165810168-363fb9e8-735f-41d5-8c1f-692dfcf153c9.jpeg)

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.63.0 (66585841)
Device: iPhone13,4 (iOS 15.4.1)
</details>

**Comment by stevepopovich** [comment, marked as answer]
Hi there, thanks for the feedback! Gotcha, so `Z-TeamDelight` should be 1 label. We should definitely not break these up. I will pass this on to the team to get it fixed, thanks!

**Reply by ghost** [reply]
Right 🙃
And sorry, I now realize my feedback could have been mentioned the actually broken label, oops.

**Reply by stevepopovich** [reply]
No worries, it was clear enough, just wanted to confirm!
